src/System.Security.AccessControl/src/System.Security.AccessControl.csproj

```diff
@@ -120,6 +120,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.Collections" />
+    <Reference Include="System.Collections.NonGeneric" />
```

Comment:
That's what confused me -- the ref is built for netstandard. Why's that? Do we just do that for convenience whenever it happens to compile successfully for netstandard?